mod/profile/views/default/profile/css.php

```diff
@@ -59,12 +59,7 @@
 #profile-details .wire-status {
 	margin-top: 10px;
 }
-#profile-details .odd {
-	border-bottom: 1px solid #DCDCDC;
-	margin: 0;
-	padding: 5px 0;
-}
-#profile-details .even {
+.profile-fields > div {
```

Comment:
can also do a .elgg-divider-bottom + .pvs then there will be no need for the extra div and classes